apps/qubit/modules/informationobject/actions/renameAction.class.php

```diff
@@ -37,55 +37,75 @@ public function execute($request)
       return sfView::NONE;
     }
 
-    $resource = $this->updateFields();
-
-    // Let user know description was updated (and if slug had to be adjusted)
+    // Internationalization needed for flash messages
     ProjectConfiguration::getActive()->loadHelpers('I18N');
 
-    $message = __('Description updated.');
+    // Handle rename form submission
+    if (null !== $request->rename)
```

Comment:
You probably wanted to use here something like ```if ($request->isMethod('post')) { ... }```, e.g.: https://github.com/artefactual/atom/blob/qa/2.3.x/apps/qubit/modules/informationobject/actions/editAction.class.php#L617-L628